CODEOWNERS

```diff
@@ -1 +1 @@
-* @hashicorp/packer @stvnjacobs @charliekenney23 @phillc 
+* @lgarber-akamai
```

Comment:
There's a warning with this change according to Github, I presume since @lgarber-akamai does not have `owner` access to this repo yet?

Reply:
Great question, since Lean (@lgarber-akamai) is not in the HashiCorp org we see this warning. Once the repo is transferred into its final location the warning should disappear.

Reply:
Perfect, thanks for the precision here!